### PR TITLE
[FLINK-33777] Fix ParquetTimestampITCase>FsStreamingSinkITCaseBase failing

### PR DESCRIPTION
**Pull request by snuyanzin**
## What is the purpose of the change

There is a build failure[1] which is a result merging this https://github.com/apache/flink/pull/23885
however it was based on a commit before https://github.com/apache/flink/pull/18304 and didn't take into account that changes...

[1]  https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=55334&view=logs&j=2e8cb2f7-b2d3-5c62-9c05-cd756d33a819&t=2dd510a3-5041-5201-6dc3-54d310f68906&l=12272

## Verifying this change

This change is already covered by existing tests _ParquetTimestampITCase_

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): ( no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no)
  - The serializers: ( no)
  - The runtime per-record code paths (performance sensitive): ( no )
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: ( no)
  - The S3 file system connector: ( no)

## Documentation

  - Does this pull request introduce a new feature? ( no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

File: flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/FsStreamingSinkITCaseBase.scala
Comment (dawidwys) [inline flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/FsStreamingSinkITCaseBase.scala:29]: Is this import used?

**Review by dawidwys — APPROVED** [review]
Thanks @snuyanzin for fixing the CI! LGTM, once this is merged we can take care of the next CI issue which is also caused by the merge order :( https://github.com/apache/flink/pull/23893

File: flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/FsStreamingSinkITCaseBase.scala
Comment (snuyanzin) [reply flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/FsStreamingSinkITCaseBase.scala:29]: Good catch
I 've again forgot that spotless does not clean unused imports in scala...
I will add commit to clean it once `test_ci connect` is finished on ci to be sure that the issue fixed

Comment (dawidwys) [reply flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/FsStreamingSinkITCaseBase.scala:29]: :( now we're in a situation, we need to squash the two fixes 🤦

Comment (snuyanzin) [reply flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/FsStreamingSinkITCaseBase.scala:29]: could it be cherry-picked to one of the PRs and merged nothe commits 
or you mean squash both fixes in one commit?

Comment (dawidwys) [reply flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/FsStreamingSinkITCaseBase.scala]: > could it be cherry-picked to one of the PRs and merge the commits
or you mean squash both fixes in one commit?

separate commits, absolutely. I meant to squash PRs 😅

**Comment by dawidwys** [comment]
I pushed your changes @snuyanzin  to https://github.com/apache/flink/pull/23893

**Comment by snuyanzin** [comment]
closing since it was merged as part of https://github.com/apache/flink/pull/23893

**Pull request closed without merging**
